Makefile

```diff
@@ -14,9 +14,30 @@ compile:
 doRelease:
-	./gradlew bintrayUpload -PbintrayUser="$(BINTRAY_USERNAME)" -PbintrayKey="$(BINTRAY_KEY)" -PmavenCentralUser="$(MAVEN_USER)" -PmavenCentralPassword="$(MAVEN_PASS)" -PmavenCentralSync=true -PdryRun=false --info
+	# bintray/jcenter upload
+	./gradlew bintrayUpload -PbintrayUser="$(BINTRAY_USERNAME)" -PbintrayKey="$(BINTRAY_KEY)" -PdryRun=false --info
+
+	# upload symbols
+	cd sentry-android-ndk
+	# be sure SymbolCollector.Console is in your $PATH - https://github.com/getsentry/symbol-collector/releases
+	SymbolCollector.Console --upload directory --path ./ --batch-type android --bundle-id sentry-android-ndk-$VERSION --server-endpoint https://symbol-collector.services.sentry.io
+	cd ..
+
+	# maven central sync using jfrog cli - https://jfrog.com/getcli/
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-log4j2/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-logback/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-servlet/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-spring/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-apache-http-client-5/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-spring-boot-starter/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-java/io.sentry:sentry-jul/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-android/io.sentry:sentry-android/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-android/io.sentry:sentry-android-core/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-android/io.sentry:sentry-android-ndk/$VERSION
+	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS getsentry/sentry-android/io.sentry:sentry-android-timber/$VERSION
```

Comment:
`l`: Maybe something like this would make it a bit easier to maintain.

```suggestion
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-log4j2/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-logback/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-servlet/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-spring/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-apache-http-client-5/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-spring-boot-starter/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-java/io.sentry:sentry-jul/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-android/io.sentry:sentry-android/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-android/io.sentry:sentry-android-core/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-android/io.sentry:sentry-android-ndk/$(VERSION)
	make upload-to-jfrog url=getsentry/sentry-android/io.sentry:sentry-android-timber/$(VERSION)

upload-to-jfrog:
	jfrog bt mcs --user=$BINTRAY_USERNAME --key=$BINTRAY_KEY --sonatype-username=$MAVEN_USER --sonatype-password=$MAVEN_PASS $(url)
```

Reply:
this PR is probably gonna be closed due to recent announcement, bintray and jcenter are gonna be retired.

Reply:
Ah, that's true.